src/main/resources/assets/bcadditions/lang/sk_SK.lang

```diff
@@ -11,66 +11,429 @@ tile.blockDusterKinetic.name=Kinetický drvič
+material.ignatius.name=Ignatius
+material.shadowIron.name=Tieňové železo
+material.lemurite.name=Lemurit
+material.midasium.name=Midasium
+material.vyroxeres.name=Vyroxeres
+material.ceruclase.name=Ceruclas
+material.alduorite.name=Alduorit
+material.kalendrite.name=Kalendrit
+material.vulcanite.name=Vulkanit
+material.sanguinite.name=Sanguinit
+material.shadowSteel.name=Tieňová oceĽ
+material.inolashite.name=Inolashit
+material.amordrine.name=Amordrin
+material.zinc.name=Zinok
+material.brass.name=Mosadz
+material.electrum.name=Elektrum
+material.aluminum.name=Hliník
+material.ardite.name=Ardit
+material.manyullyn.name=Manyullyn
+material.cobalt.name=Kobalt
+material.copper.name=Meď
+material.lead.name=Olovo
+material.nickel.name=Nikel
+material.platinum.name=Platina
+material.silver.name=Striebro
+material.tin.name=Cín
+material.apatite.name=Apatit
+material.osmium.name=Osmium
+material.sulfur.name=Síra
+material.saltpeter.name=Liadok
+material.certusQuartz.name=Kremeň
+material.aluminumBrass.name=Hliníková mosadz
+material.alumite.name=Alumit
+material.pigIron.name=Prasacie železo
+material.invar.name=Invar
+material.signalum.name=Signálum
+material.lumium.name=Lumium
+material.enderiumBase.name=Základ endéria
+material.enderium.name=Endérium
+material.electricalSteel.name=Electrická oceľ
+material.energeticAlloy.name=Energetická zliatina
+material.phasedGold.name=Zlato
+material.redstoneAlloy.name=Ruditová zliatina
+material.conductiveIron.name=Vodivé železo
+material.phasedIron.name=Železo
+material.darkSteel.name=Temná oceľ
+material.soularium.name=Dušarium
+material.fzDarkIron.name=Temné železo
+material.gildedRedMetal.name=Pozlátený červený kov
+material.yellorium.name=Yelórium
+material.graphite.name=Grafit
+material.cyanite.name=Kyanid
+material.blutonium.name=Blutónium
+material.ludicrite.name=Ludikrit
+material.pyrotheum.name=Pyroteum
+material.cryotheum.name=Kryoteum
+material.blizz.name=Ľadovec
+
+entity.bcadditions.bcaLaserShot=Lasér
+
+death.attack.bcaLaser=%2$s spálil lasérom %1$s
+death.attack.bcaLaser.item=%2$s bol spálený hráčom %1$s pomocou %3$s
+
+tooltip.installed=Naištalované vylepšenia:
 tooltip.deactivated=Deaktivované
 tooltip.upgradesPossible=Možné vylepšenia
+tooltip.holdShiftForMoreInfo=Drž shift pre viac informácií
+tooltip.FluidCapacity=Kapacita: %s vedier
+tooltip.storedFluid=Práve je tu uložených %s mB z %s mB
+tooltip.colorNormalMode=Normálny mód
+tooltip.colorNormalMode.info=Zafarbí bežné farbiteľné predmety, ovce a BC rúry
+tooltip.colorSortingMode=Triediaci mód
+tooltip.colorSortingMode.info=Zafarbí triediace časti diazuritovej a lazuritovej rúry.
+tooltip.defaultColor=Štandardná farba
+tooltip.compressor.fill=Naplniť
+tooltip.compressor.empty=Vyprázdniť
+tooltip.none=Žiadne
+tooltip.forKineticMultiTool=Použiteľné s kinetickým multinástrojom
+tooltip.forUpgradableMachines=Použiteľné s prístrojmi, ktoré sa dajú vylepšiť
+tooltip.stick.0=Najlepšie funguje %s čipom
+tooltip.stick.1=%s tyč pre inštaláciu
```

Comment:
I had to be a bit creative here, because my language has double negation. But it works.